docs/backend-api/resources/tracking/tracker/rules/rule_types.md

```diff
@@ -0,0 +1,1332 @@
+---
+title: Rule types
+description: Rule types
+---
+
+# Rule types
+
+Rule types with all parameters to create. The rule availability depends on the device and rule integration for it.
+
+### Geofence entrance or exit
+
+A rule that triggers on entering/exiting geofences.
+
+#### parameters

```

Comment:
So much copy&paste... Extract common parameters to one table, plase.

Reply:
> Ou, commits from the last pull request are here too.

Just close prevous review

Reply:
will change the file and commit
closed another request

Reply:
commited